style.css

```diff
@@ -1,8 +1,409 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  border: none;
+  outline: none;
+  scroll-behavior: smooth;
+  font-family: "lato", sans-serif;
+}
+
+html {
+  line-height: 1.15;
+  -webkit-text-size-adjust: 100%;
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
```

Comment:
You still need to fix it. please see the comparison in following

https://github.com/ibtisam-adil/Capstone/assets/118625824/538a3038-27fa-4100-9174-5466d97a13c4